lib/rules/disallow-comma-before-line-break.js

```diff
@@ -51,12 +90,47 @@ module.exports.prototype = {
     },
 
     check: function(file, errors) {
+        var exceptFunction = this._exceptFunction;
+        var lineBreak = this._lineBreak;
+
+        function canSkip(token) {
+            if (!exceptFunction) {
+                return false;
+            }
+            var node = file.getNodeByRange(token.range[0]);
+
+            if (node.type !== 'ObjectExpression' ||
```

Comment:
This expression verifies that we check only multiline objects with more that one property:
* `allExcept: ['function']` does lookup only inside ObjectExpression
* No point checking singleline objects for line breaks
* No comma if there's only a single property